lib/cisco_node_utils/bgp_af.rb

```diff
@@ -226,6 +226,22 @@ def default_additional_paths_selection
       config_get_default('bgp_af', 'additional_paths_selection')
     end
 
+    # advertise_l2vpn_evpn
```

Comment:
TestRouterBgpAF#test_redistribute = 17.61 s = .
TestRouterBgpAF#test_advertise_l2vpn_evpn = 7.07 s = .
TestRouterBgpAF#test_maximum_paths = 9.07 s = .
TestRouterBgpAF#test_default_information_originate = 5.95 s = .
TestRouterBgpAF#test_collection_empty = 3.22 s = .
TestRouterBgpAF#test_utils_delta_add_remove = 2.70 s = .
TestRouterBgpAF#test_additional_paths_selection = 5.72 s = .
TestRouterBgpAF#test_dampening = 8.54 s = .
TestRouterBgpAF#test_additional_paths = 6.59 s = .
TestRouterBgpAF#test_next_hop_route_map = 6.33 s = .
TestRouterBgpAF#test_maximum_paths_ibgp = 7.88 s = .
TestRouterBgpAF#test_dampen_igp_metric = 6.38 s = .
TestRouterBgpAF#test_network = 18.38 s = .
TestRouterBgpAF#test_client_to_client = 6.26 s = .
TestRouterBgpAF#test_collection_not_empty = 6.03 s = .

Finished in 117.732287s, 0.1274 runs/s, 2.0980 assertions/s.